src/main/java/roomescape/dto/ReservationRequest.java

```diff
@@ -12,32 +12,25 @@ public class ReservationRequest {
     private String date;
     private String time;
     private String name;
+    private Long theme_id;
```

Comment:
자바에서는 스네이크 케이스 대신 대부분 카멜 케이스를 표준으로 따르고 있습니다! :)

```suggestion
    private Long themeId;
```